### PR TITLE
DOCS-941: Fix table rendering in includes

### PR DESCRIPTION
**Pull request by andf-viam**
Fix table rendering in includes by removing wrapping `<div>` elements from `readfile.html`.

- Also using opportunity to expand `note.md` to showcase HTML and MD tables in includes, useful for testing this.

### CODE REVIEW

File: docs/note.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ Figure styles the Attribution text as body text.
 
 {{< readfile "/static/include/sample.md" >}}
 
-Section content before this line is contained in an included file: /static/include/sample.md
+Section content before this line is contained in an included file: <file>/static/include/sample.md</file>
```
Comment (JessamyT) [inline docs/note.md:270]: Why this change?

File: static/include/sample.md [hunk 2]
```diff
@@ -1,24 +1,94 @@
-
 ### This is a Heading in the Sample File
-This entire section was added to the current topic using the readfile shortcode.
+
+This entire section was added to the current topic using the `readfile` shortcode.
 
 {{% alert="Note" color="note" %}}
```
Comment (JessamyT) [inline static/include/sample.md:5]:
```suggestion
{{% alert="Important" color="note" %}}
```
[out of diff nit] While we're here...realizing we don't use "note" anymore so should be consistent through this doc :)

File: static/include/sample.md [hunk 3]
```diff
@@ -1,24 +1,94 @@
...
-Changes to included files do not appear in local mode (hugo server -D). You'll need to halt and restart the server to view the changes.
+Changes to included files do not appear in local mode (`hugo server -D`). You'll need to halt and restart the server to view the changes.
```
Comment (JessamyT) [inline static/include/sample.md:6]:
```suggestion
Changes to included files do not appear in local mode (`hugo server -D`).
You'll need to halt and restart the server to view the changes.
```
[out of diff opt nit]

**Review by JessamyT — APPROVED** [review]
Looks good function-wise, though I still don't know why the fence is there :) so maybe I shouldn't be reviewing. Threw some handy out-of-diff nits in there too; can of course ignore. Not sure why the change on line 270 of note.md though

File: docs/note.md (hunk 1 above)
Comment (andf-viam) [reply docs/note.md:270]: Why not this change? It's a file path!
